nlgeval/tests/test_nlgeval.py

```diff
@@ -118,4 +118,4 @@ def test_compute_metrics(self):
         self.assertAlmostEqual(0.88469, scores['EmbeddingAverageCosineSimilairty'], places=5)
         self.assertAlmostEqual(0.568696, scores['VectorExtremaCosineSimilarity'], places=5)
         self.assertAlmostEqual(0.784205, scores['GreedyMatchingScore'], places=5)
```

Comment:
Thanks for the contribution!
Would you add some tests for the value of the ROUGE metrics?

Reply:
Sorry, but I will not :

* The code added is from another repository, not my code
* The score are slightly different with pyrouge. 
* The goal of this PR is to give a quick and approximate way to get ROUGE-N score. This should not be merged to main branch, but kept open here.
* For a real ROUGE-N score, someone needs to add the official perl script ROUGE-155... I don't have time for this now :/

Reply:
The other repo's code seems to be Apache-licensed, I'm not sure if we can merge it, particularly without including their license. I'm not too worried about slightly different values as long as we're clear about the methods used in the docs. Are you aware where differences might come from?

Reply:
We could at least test that the values are within some reasonable bounds.

Reply:
I tried comparing the results with the results using the [rouge package](https://github.com/pltrdy/rouge). But scores are different (and not only a few digits...). Not only ROUGE-N are different, but also the existing ROUGE-L.

This package is also Apache-licensed. Not sure if we can just use it without including their license (without modifying their code).